raiden-dapp/src/services/raiden-service.ts

```diff
@@ -669,7 +670,11 @@ export default class RaidenService {
     if (!raiden.mainAddress) {
       return [];
     }
-    const allTokens = await raiden.getTokenList();
+    const allTokenAddresses = uniq(
+      (await raiden.getTokenList()).concat(
```

Comment:
Can you get this list from somewhere else? Maybe `store.getters.tokens.map(token => token.address)`. `getTokenList` scans the blockchain, so it's slow, and since this was already done to fetch the tokens on initialization, you may just populate it from there.

Reply:
Ah, ok I see.

Reply:
`store.getters.tokens` just returns an empty array. But I removed that code from the `raiden-service` and I do all the work in the `Withdrawal.vue` component now.